### ISSUE TITLE
Can I drop the puppeteer version from 20 to 19 in the latest initialized `stencil` project?

### ISSUE THREAD

**Issue by ayangweb** [ionic-team/stencil#4646]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already include this feature request, without success.

### Describe the Feature Request

As I mentioned in #4634, can I just downgrade to version 19?

### Describe the Use Case

As I mentioned in #4634, can I just downgrade to version 19?

**Comment by rwaskiewicz** [comment]
>  I tested it and installing v19 gives no errors and the unit tests work fine, I'm not sure if it's being used for anything else, so can I just downgrade to v19

Yes, Stencil supports Puppeteer v19 (assuming you are using Stencil v3 or newer).

Please note that the issue tracker is meant to be used exclusively for bug reports/feature requests. We kindly ask that you direct support questions such as these to either the Stencil [Discord](https://chat.stenciljs.com) or the [Ionic Forums](https://forum.ionicframework.com/c/stencil/21). Thanks!
